app/_how-tos/generate-a-collection-from-an-api-specification.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Generate a collection from an API specification
+
+products:
+    - insomnia
+
+tags:
+  - collections
+  - documents
+
+tldr: 
+  q: How do I generate a collection from an API spec?
+  a: Create or import an API spec and define environment variables, then click cog icon and click Generate collection.
```

Comment:
```suggestion
  a: Create or import an API spec and define environment variables, then click the Settings icon and click **Generate collection**. Generating a collection saves more time than creating one manually and you can ensure that all requests are consistent with the API specification, reducing errors.
```
Just wanted to add that suggestion because I think for me, I didn't understand the why behind generating vs just plain creating a collection.

Reply:
I think it's more document vs collection than generating vs creating a collection. Importing a spec as a collection would take about as much time as importing a spec as a document and then generating a collection. I think the main difference is that when you create a document you have access to design features, you can edit the spec and create unit tests, which you can't do from a standalone collection (if I understand correctly). So to me they're different use cases, one is design and the other is just sending requests. I'll update the how-to to make it clearer that we're talking about design documents.